.github/workflows/label_checker.py

```
@@ -1,29 +1,46 @@
-component_label = "Component/" + component
-if component_label in all_existing_labels:
-    labels.append(component_label)
-
-# Get component version
-version = os.environ["ISSUE_BODY"][j+13:j+18].strip()
-affected_label = "Affected/" + component + "-" + version
-if affected_label in all_existing_labels:
-    labels.append(affected_label)
+page = 1
+while True:
+    response = requests.get(
```

Ruff (0.16.0)

[error] Probable use of `requests` call without timeout

(S113)